src/tests/user.rs

```diff
@@ -262,13 +267,9 @@ fn user_total_downloads() {
     let mut req = req(Method::Get, &format!("/api/v1/users/{}/stats", u.id));
     let mut response = ok_resp!(middle.call(&mut req));
 
-    #[derive(Deserialize)]
-    struct Response {
-        total_downloads: i64,
-    }
-    let response: Response = ::json(&mut response);
-    assert_eq!(response.total_downloads, 30);
-    assert!(response.total_downloads != 32);
+    let stats: UserStats = ::json(&mut response);
+    assert_eq!(stats.total_downloads, 30);
+    assert!(stats.total_downloads != 32);
```

Comment:
o.O i know you didn't do this, but we're asserting that `total_downloads` is 30 and then asserting that it's not 32????